oam/install_scripts/columnstore-post-install.in

```diff
@@ -177,13 +177,17 @@ if [ -z "aws" ]; then
 fi
 
 postConfigure
-systemctl start mariadb-columnstore
 
-# Wait for all columnstore to be ready, DDLProc is final process in startup order
-while  [ -z  "$(pgrep -x DDLProc)" ];
-do
-    sleep 1
-done
+systemctl cat mariadb-columnstore.service > /dev/null 2>&1
+if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
```

Comment:
We shouldn't be starting a non-existent systemd unit so let's include this check.

In the future we may have to add an else to take into account what we want to happen on Sky, but for tomorrow's release this is sufficient.